community-plugins.json

```diff
@@ -6324,7 +6324,7 @@
         "id": "personal-assistant",
         "name": "Personal Assistant",
         "author": "edony",
-        "description": "Streamline workflows within Obsidian, managing memos, formatted records, callouts, frontmatter, local graph, themes and plugins with just one command.",
+        "description": "AI-powered workflows in Obsidian streamline the management of records, callouts, frontmatter, graph view, themes, and plugins.",
```

Comment:
Please remove "Obsidian" from the description, it should be obvious that this a plugin for Obsidian.

Reply:
Thank you for your comment. I have already updated the description to omit 'Obsidian,' as it is self-evident and unnecessary to repeat.